src/pages/BuildFeed.vue

```diff
@@ -73,31 +73,26 @@
         }
       },
       updateFilter () {
-        this.$store.dispatch('buildsFeed/updateFilter', this.queryToFilter(this.query))
+        this.$store.dispatch('buildsFeed/updateFilter', this.queryToFilter(this.$route.query))
+      },
+      toBool (value) {
+        let bool = null
```

Comment:
Question, does it make a difference if value is `null`?
If not, why not just
```
return a === 'true'? true: false
```
Up to you, it's just my personal taste

Reply:
We can set third state for this toggle. For example, we can choose not signed and signed builds. I am unsure if it is right but I decided to keep it.

Reply:
please change default value for `bool` to `false`, because we want to return boolean value